app/models/procedure_presentation.rb

```diff
@@ -1,3 +1,45 @@
 class ProcedurePresentation < ApplicationRecord
   belongs_to :assign_to
+
+  delegate :procedure, to: :assign_to
+
+  validate :valid_displayed_fields
+  validate :valid_sort_column
+  validate :valid_filter_columns
```

Comment:
Pour `:valid_displayed_fields`, j'ai du mal à parser si c'est une question ("is it valid?") ou une action ("do the validation").

Mais j'ai l'impression que c'est une action. Est-ce que renommer avec un verbe (`:validate_displayed_fields`, ou `:check_allowed_displayed_fields`, qqc comme ça) pourrait aider ?

Reply:
La version en `check_allowed` se lit très bien je trouve